PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/Util.java

```diff
@@ -466,6 +503,31 @@ private static void addReservedKeys(String key) {
         }
     }
 
+    private static void handleMoPubBuilderCustomTargeting(HashMap<String, String> bids, Object requestParametersBuilder) {
+        Object requestParameters = Util.callMethodOnObject(requestParametersBuilder, "build");
```

Comment:
Why do you call build here? Is there not another way to remove the used keywords?

Reply:
Just followed how it was done for GAM. But removed it, as I don't see if it serves any purpose.